Classes/Controller/ProductController.php

```diff
@@ -126,9 +126,9 @@ protected function isActionAllowed(string $action): bool
     {
         $frameworkConfiguration = $this->configurationManager->getConfiguration($this->configurationManager::CONFIGURATION_TYPE_FRAMEWORK);
         // @extensionScannerIgnoreLine
```

Comment:
Can we remove this line or check if it is necessary? No idea, why this could be a false positive, so it has to be ignored.

Reply:
Done. I made a forced push to not add another commit.